holidays/countries/suriname.py

```diff
@@ -0,0 +1,258 @@
+    InternationalHolidays,
+    IslamicHolidays,
+)
+from holidays.observed_holiday_base import ObservedHolidayBase, SAT_SUN_TO_NEXT_WORKDAY
+
+
+class Suriname(
+    ObservedHolidayBase,
+    ChineseCalendarHolidays,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+):
+    """Suriname holidays.
+
+    References:
+        * [Besluit Vrije Dagen 1971](https://web.archive.org/web/20250411201210/https://www.sris.sr/wp-content/uploads/2022/10/Besluit-Vrije-dagen-GB-1971-no-78.pdf)
+        * https://web.archive.org/web/20250411203621/https://gov.sr/wp-content/uploads/2022/09/arbeidswet.pdf
+        * [S.B. 2007 no. 98](https://web.archive.org/web/20250411204836/https://www.sris.sr/wp-content/uploads/2021/10/SB-2007-no-98-besluit-vrije-dagen.pdf)
+        * [S.B. 2012 no. 21](https://web.archive.org/web/20250411204754/https://www.sris.sr/wp-content/uploads/2021/10/SB-2012-no-21-besluit-vrije-dagen.pdf)
+        * [S.B. 2021 no. 27](https://web.archive.org/web/20250411210516/https://www.sris.sr/wp-content/uploads/2021/03/S.B.-2021-no.-27-houdende-nadere-wijziging-van-het-Besluit-Vrije-Dagen-1971.pdf)
+        * [S.B. 2024 no. 167](https://web.archive.org/web/20250411211129/https://www.sris.sr/wp-content/uploads/2025/01/S.B.-2024-no.-167-Algemene-termijnenwet.pdf)
+
+    Note:
+        The oldest decree available online that underpins the public holidays defined here
+        for Suriname is Besluit Vrije Dagen 1971 of April 22, 1971.
+    """
+
+    country = "SR"
+    default_language = "nl"
+    # %s (estimated).
+    estimated_label = tr("%s (geschat)")
+    # Day after the %s.
+    observed_label = tr("Dag na de %s")
+    # Day after the %s (estimated).
+    observed_estimated_label = tr("Dag na de %s (geschat)")
+    start_year = 1972
+    supported_languages = ("en_US", "nl")
+
+    def __init__(self, islamic_show_estimated: bool = False, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChineseCalendarHolidays.__init__(self)
+        ChristianHolidays.__init__(self)
+        HinduCalendarHolidays.__init__(self, cls=SurinameHinduHolidays)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=SurinameIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_WORKDAY)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        dt = self._add_new_years_day(tr("Nieuwjaarsdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if 1981 <= self._year <= 1992 or 2012 <= self._year <= 2020:
+            # Day of Liberation and Renewal.
+            self._add_holiday_feb_25(tr("Dag van Bevrijding en Venieuwing"))
```

Comment:
```suggestion
            self._add_holiday_feb_25(tr("Dag van Bevrijding en Vernieuwing"))
```
Official sources also contain typos.